src/typings/decoders.ts

```diff
@@ -32,7 +32,13 @@ interface DecoderOptions {
  * @property level
  * @property number
  */
-type DecodeResult = [string, bigint, number, number];
+type DecodeResult = {
```

Comment:
sorry for missing this in the last round
```
/**
 * @property logEventNum The log event's 1-based index in the log file.
 * @property logLevel The log event's log level as a number (maps to enum LOG_LEVEL).
 * @property message The log event's message.
 * @property timestamp The log event's timestamp in milliseconds since the Unix epoch.
 * @property utcOffset The log event's local time zone offset from UTC, in minutes.
 */
interface DecodeResult {
...
}
```